blog-server-services/src/traits/post_service.rs

```diff
@@ -5,6 +5,7 @@ use serde::{Deserialize, Serialize};
 #[serde(rename_all = "snake_case")]
 pub struct Tag {
     pub id: i64,
```

Comment:
u64 🤡

Reply:
changed, also for some methods and other entities